frontend/prisma/schema.prisma

```diff
@@ -25,4 +25,17 @@ model Answers {
   calculatorId String
   answers      Json
   matches      Json
+  userId       String?
+  User         User?         @relation(fields: [userId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+}
+
+model User {
+  id             String    @id @default(uuid())
+  name           String?   @db.String(255)
+  email          String    @unique @db.String(255)
+  authProvider   String?   @db.String(32)
+  authProviderId String?   @db.String(255)
+  createdAt      DateTime  @default(now()) @db.Timestamptz()
+  updatedAt      DateTime  @updatedAt @db.Timestamptz()
+  Result         Answers[]
```

Comment:
Zatím jen přemýšlím – abychom byli (datově) ready na rozvyplňovaných víc (stejných) kalkulaček, podle mě potřebujeme nějakou další párovací tabulku.

Reply:
to snad nevadí, ne? spíš koukám, proč se to tady tak blbě jmenuje, mělo by to být `Answers         Answers[]`

Reply:
No, neuděláš dotaz „dej mi všechny odpovědi na kalkulačku X pro uživatele X“, to by se muselo filtrovat až pak mimo DB (nebo by se tam musel přidat index na calculatorID, což @martinwenisch nechce).

Reply:
Me prijde ok mit jediny dva dotazy "dej mi kalkulacku ID" a "dej mi vsecky kalkulacky pro uzivatele X" (tedy index na `id` a `userId`). Vsecko ostatni muzeme dopocitat aplikacne. Tahle tabulka bude vyrazne vetsi nez vsecko ostatni, tak ji netreba zatezovat.

Reply:
Asi jo. Ale ten druhý index netřeba, to si uděláš dotaz na Users a pak si můžeš už vyfiltrovat podle Answers podle ID.

Reply:
zmenil jsem to `Answers         Answers[]` a jaky je tedy vystup? mam neco upravovat?